### PR TITLE
Auto update README when publishing to Docker Hub

### LINKED ISSUE #17: Automatically update README when publishing to Docker Hub
README on Docker Hub currently needs to be updated manually. It would be nice if README would be automatically updated every time a new image is pushed to Docker Hub

### PR DESCRIPTION
**Pull request by grc0**
Updates README on Docker Hub when publishing a release.

Fixes #17

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -70,3 +70,12 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+      # Update Readme on Docker Hub
+      - name: Docker Hub Description
+        if: github.event_name != 'pull_request'
+        uses: peter-evans/dockerhub-description@v3
```
Comment (pascalberger) [inline .github/workflows/publish.yml]: Pin to specific version to have reproducible builds. Renovate will taking care of updating actions afterwards.

```suggestion
        uses: peter-evans/dockerhub-description@14881160433a81f9ebc838a253d4edd9e8fc35f1 # tag=v3.1.1
```

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -70,3 +70,12 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+      # Update Readme on Docker Hub
+      - name: Docker Hub Description
```
Comment (pascalberger) [inline .github/workflows/publish.yml]:
```suggestion
      - name: Publish README to Docker Hub
```

**Review by pascalberger — CHANGES_REQUESTED** [review]
Two smaller issues. Otherwise LGTM

File: .github/workflows/publish.yml (hunk 2 above)
Comment (grc0) [reply .github/workflows/publish.yml]: @pascalberger Updated the RP with your feedback.

**Review by pascalberger — APPROVED** [review]
LGTM

**Comment by pascalberger** [comment]
@grc0 your changes have been merged, thanks for your contribution 👍

**Pull request merged**